src/app/components/graph/graph-viz.js

```diff
@@ -206,6 +206,20 @@ angular
                     },
                     show: true,
                 },
+                {
+                    id: 'export-png',
+                    content: 'Export PNG',
+                    selector: 'node',
```

Comment:
I think you can add
```
coreAsWell: true,
```

to add the `Export PNG` option to the context menu when a user right-clicks on the background of the dag too.